### PR TITLE
completed /api/categories feature and updated README

### PR DESCRIPTION
**Pull request by a-stenhouse**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+const app = express();
+const { getCategories } = require("./controller.js")
+
+app.use(express.json());
+
+
+app.get("/api/categories", getCategories)
+
+app.use((err, req, res, next) => {
+    console.log(err);
```
Comment (JVal-NC) [inline app.js]: Remove console.log please

File: app.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const app = express();
+const { getCategories } = require("./controller.js")
+
+app.use(express.json());
```
Comment (JVal-NC) [inline app.js]: Not needed for this ticket, please remove

File: __tests__/app.test.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+});
+
+describe("get /api/categories", () => {
+    test("responds with JSON containing an array of category objects with a key of 'categories'", () => {
+        return request(app)
+            .get("/api/categories")
+            .expect(200)
+            .then((res) => {
+                expect(res.body).toEqual({
+                    categories: [
```
Comment (JVal-NC) [inline __tests__/app.test.js]: Test could be more generic, use jest matchers to outline the object keys and expected value data types and can then use a length property to check number of entries returned. Have a look at toMatchObject

**Review by JVal-NC — COMMENTED** [review]
Good start, main comment regarding your test to help with future testing. Let me know if your unsure how to approach it.

**Review by Keb84 — APPROVED** [review]
Changes look good 🙂

**Pull request merged**